.env.template

```diff
@@ -26,7 +26,7 @@ JUMIO_URL=amer-1.jumio.ai/api/v1
 JUMIO_WORKFLOW_DEFINITION=10013
 MAGIC_SECRET_KEY=test
 MAX_DEPOSIT_SIZE=0.6
-MAX_KYC_ATTEMPTS=3
+KYC_MAX_ATTEMPTS=3
```

Comment:
This change makes sense to me since the validation expects this key

Reply:
yeah, that's why i did it. you get a rejection if you use the template copy